package.json

```diff
@@ -97,17 +108,23 @@
     "parse-json": "^4.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "7.4.4",
+    "@babel/core": "7.4.5",
+    "@babel/plugin-proposal-class-properties": "7.4.4",
+    "@babel/preset-env": "7.4.5",
+    "@babel/preset-flow": "7.0.0",
     "babel-eslint": "^10.0.1",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "codecov": "3.5.0",
```

Comment:
Why does the way we deal with codecov need to change at all?

Reply:
I tried to put the why in the commit: _"codecov automatically merges tests from multiple places for the same commit"_. This includes windows and all node tests. Not necessary at all. Should I remove the commit?

Reply:
I'm not sure I understand that sentence as an explanation. Would you mind restating?

Reply:
If code ever gets added such as:

```js
import { platform } from 'os';

if (platform() === 'win32') {
    // do windows stuff
}

if (process.version === '8') {
    // do node 8 stuff
}
```

You'll keep 100% code coverage without having to ignore the lines. This is probably unlikely though and could be dealt with if it ever comes up.

Reply:
> If code ever gets added such as ... This is probably unlikely though and could be dealt with if it ever comes up.

Yeah, let's deal with it if/when it comes up.

Reply:
Removed